objects/LuaScriptState.luascriptstate

```diff
@@ -1 +1 @@
-{"optionPanel":{"showChaosBagManager":false,"showCleanUpHelper":false,"showDrawButton":false,"showHandHelper":{},"showNavigationOverlay":false,"showTokenArranger":false,"useClueClickers":false,"useSnapTags":true}}
+{"optionPanel":{"showChaosBagManager":false,"showCleanUpHelper":false,"showDrawButton":false,"showHandHelper":[],"showNavigationOverlay":false,"showTokenArranger":false,"useClueClickers":false,"useSnapTags":true}}
```

Comment:
Err, no idea honestly.  Will fix directly and look into what happened.

Reply:
Ah, I see - I think this is because the hand helper option panel is an array now, not a single value.  Not sure why that didn't commit before, but it should be correct.